api_handlers.go

```diff
@@ -165,7 +166,12 @@ func apiTestRunPostHandler(w http.ResponseWriter, r *http.Request) {
 		http.Error(w, "Failed to parse JSON: " + err.Error(), http.StatusBadRequest)
 		return
 	}
-	testRun.CreatedAt = time.Now()
+
+	// Use 'now' as created time, unless flagged as retroactive.
+	if retro, err := strconv.ParseBool(r.URL.Query().Get("retroactive"));
+	    err != nil || !retro {
```

Comment:
Is this indentation pattern standard for Go?

Reply:
Apparently not, gofmt'd